docs/releasing.md

```diff
@@ -162,10 +151,6 @@ echo "Complete the _Release publicity_ tasks documented above"
 
 ### Chromium CL
 
-If this is a branching week, wait until _after_ the branch point email, then land the CL.
```

Comment:
😢

Reply:
oh, you're not in this group chat. no one responded, so.. 
![image](https://user-images.githubusercontent.com/4071474/94841444-a671d380-03df-11eb-8102-4e6de2757a89.png)